client/app/components/Input/InputTag.jsx

```diff
@@ -29,6 +29,13 @@ export class InputTag extends React.Component<Props, State> {
     this.state = { checkboxes: props.checkboxes, autoHighlight: false };
   }
 
+  static getDerivedStateFromProps(nextProps: Props, prevState: State) {
```

Comment:
So this works, but triggers extra, unwanted re-renders. We can remove this and add `key={Utils.randomString()}` to `<InputTag />` in `/client/app/widgets/Resources/index.jsx`. This trigger a re-render when the `state.checkboxes` changes (since it's passed into InputTag).